### PR TITLE
Send request metadata

### PR DESCRIPTION
**Pull request by DamirSvrtan**
I need request metadata asap and I've seen that @mootpointer did almost all the work on PR #12, I just went ahead and added:
- sending connection body params
- use format_ip to send out an ip address.

@jarednorman can you please merge this in and release a new version? Thnx a ton for the work so far!

### CODE REVIEW

**Comment by alexpusch** [comment]
This is really a must for this lib. Please merge.

File: lib/plugsnag.ex [hunk 1]
```diff
@@ -1,4 +1,6 @@
 defmodule Plugsnag do
+
```
Comment (jarednorman) [inline lib/plugsnag.ex:2]: Please don't add blank lines immediately inside blocks.

**Comment by mootpointer** [comment]
This was blocked because I wanted to work out an adequate way of filtering params. I'll circle back on that and work to merge it this week.

**Comment by mootpointer** [comment]
Looking further, I'm going to look at integrating these changes with #14 to provide the proper filtering and then look to cut a release in the coming days.

**Pull request closed without merging**